handlers.go

```diff
@@ -37,16 +37,16 @@ func (f *faucet) registerHandlers(api *apirest.API) {
 
 	if f.authTypes["oauth"] > 0 {
 		if err := api.RegisterMethod(
-			"/oauth/claim/{provider}/{code}/{to}",
-			"GET",
```

Comment:
This change from GET to POST breaks too many things (not only the frontend, also the Go SDK).
I'd instead keep both (GET + POST) and let the handler decide what to use.

Reply:
Never mind, I see you only changed the OAUTH endpoint.